File: JAVA/ReverseNumber.java
import java.util.Scanner;
public class ReverseNumber {
    public static void main(String[]args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter a number: ");
        int n = sc.nextInt(), rev=0;
        while(n!=0) {
            rev=rev*10+n%10;
            n/=10;
        }
        System.out.println("Reversed= "+rev);
    }
}

File: C/reverse_number.c
#include<stdio.h>
int main() {
    int num, reversed = 0, remainder;

    printf("Enter a number: ");
    scanf("%d", &num);

    while(num!=0) {
        remainder = num%10;
        reversed = reversed * 10 + remainder;
        num = num/10;
    }
    printf("Reversed Number = %d\n", reversed);
    return 0;
}